src/app/(auth)/(navigationsBarLayout)/mypage/SideNavigation.tsx

```
@@ -31,8 +31,19 @@ export const SideNavigation = ({ tab, setTab }: IProps) => {
+          <li
+            className={`cursor-pointer py-2 px-3 text-center rounded-md ${
+              tab === 'changePassword'
+                ? 'text-green-400 font-extrabold text-lg'
+                : 'text-white text-base font-light'
+            }`}
+            onClick={() => {
+              setTab('changePassword');
+            }}
+          >
+            비밀번호 변경
+          </li>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)